scripts/ci.sh

```diff
@@ -12,5 +18,42 @@ if [ ! "$HARDHAT_FORK_BLOCK" ]; then
   echo
 fi
 
+# TODO: skip this if it's already running
+echo "Starting Hasura on port ${CI_HASURA_PORT}..."
+
+CMD=(docker compose --profile ci up)
+
+if [ "$VERBOSE" ]; then
+  "${CMD[@]}" 2>&1 & PID=$!
+else
+  LOGFILE=${TMPDIR:-.}/hasura-ci-$(date +%s).log
+  echo "Writing output to" "$LOGFILE"
+  "${CMD[@]}" > "$LOGFILE" 2>&1 & PID=$!
+fi
+
+VERCEL_CMD=(vercel dev -t "$VERCEL_TOKEN" --confirm)
```

Comment:
can we call this `CI_VERCEL_TOKEN` to avoid confusion when it's set in the local .env file?